doc/tutorial.md

```diff
@@ -135,13 +148,18 @@ The additional parameters are:
 
 ![Script execution with adjusted __chequeLimit__](assets/polkadot.js_execute_script_buy.png)
 
-The flow of this code example is:
-* Bob: `initial_coin_minting`, but only once after publication.
-* Alice: `register_new_user`, but only once after publication.
-* Alice: `buy_coin`.
-* Alice: `wash_car` for each time you bought a washing coin.
 
-You have now successfully published a Move module on your Polkadot blockchain and executed successfully Move scripts.
+## Finishing the Tutorial
+
+You have successfully published a Move module on your Polkadot blockchain and executed a Move script successfully.
+
+You can iterate the full demonstration example for training purposes or personal interest. So far, the developer Bob has published his module for his car wash and initialized it by executing the script `initial_coin_minting`. It's time for Alice (Bob's first customer) to register, buy a coin and wash her car! Therefore, try to:
+- create transactions,
+- estimate the optimal amount of needed gas,
+- and execute them as user `Alice` for the following scripts:
+  - `register_new_user`,
+  - `buy_coin` (observe here Bob's and Alice's account balances, because the price of a washing coin is 1 UNIT),
+  - `wash_car`.
```

Comment:
```suggestion
So now we are ready to buy some coins for washing our car using our Substrate balance.

First, let's check the balance before we execute some new scripts:

<@micha: insert here a balance pallet RPC for getting Alice's funds oooor use a screenshot instead>

Now, let's execute the following actions:

1. Using Alice's account, let's register her as a customer by executing the `register_new_user.mv` script
 - Hint 1: this script requires one signer which has to be Alice's account ID.
 - Hint 2: use the estimation RPC method to calculate gas.
 - Hint 3: `cheque_amount` is unused in this script, so set it to zero.
 
2. Now, again using Alice's account, buy a coin with a `buy_coin.mv` script:
  - Hint 1: this script requires one signer which has to be Alice's account ID.
  - Hint 2: use the estimation RPC method to calculate gas.
  - Hint 1: because the price of a washing coin is 1 UNIT), we need to write a cheque of around xxxx
 
3. Wash the car with Alice's account 
  - Hint 1: ...


After the car is washed, now check Alice's balance again and see how the above operations changed her balance status:

  <@micha: insert here a balance pallet RPC for getting Alice's funds oooor use a screenshot instead>
```

Reply:
Done